UrlManager.php

```diff
@@ -134,6 +134,18 @@ class UrlManager extends BaseUrlManager
      */
     public $languageParam = 'language';
 
+    /**
+     * @var string $_SERVER key set by apache mod_geoip
+     */
+    public $geoipServerFieldName = 'HTTP_X_GEO_COUNTRY';
```

Comment:
I think a better name is `$geoIpServerVar` because most entries in `$_SERVER` are environment variables set by the webserver. We should also mention the default value in the description:

```
@var string the key in $_SERVER that contains the detected GeoIP country. Default is 
'HTTP_X_GEO_COUNTRY' as used by mod_geoip in apache.
```